proposal/proposal.tex

```diff
@@ -0,0 +1,136 @@
+\documentclass[a4paper,12pt]{article}
+
+\usepackage{geometry}
+\usepackage{hyperref}
+\geometry{margin=1.5in}
+\usepackage{enumitem}
+
+\title{Optimising \lq{matrix multiplication\rq} calculations in the tropical semiring}
```

Comment:
Give the slant towards ILP, I wonder if we need that in the title. e.g. Efficient instruction-level parallelism analysis using tropical semiring 'matrix multiplication'.